Facebook.AccountKit.iOS/source/Facebook.AccountKit/ApiDefinition.cs

```diff
@@ -0,0 +1,604 @@
+        [Export("graphVersionString")]
+        //[Verify(MethodToProperty)]
+        string GraphVersionString { get; }
+
+        // +(NSString * _Nonnull)versionString;
+        [Static]
+        [Export("versionString")]
+        //[Verify(MethodToProperty)]
+        string VersionString { get; }
+
+        // @property (readonly, copy, nonatomic) id<AKFAccessToken> _Nullable currentAccessToken;
+        [NullAllowed, Export("currentAccessToken", ArgumentSemantic.Copy)]
+        IAccessToken CurrentAccessToken { get; }
+
+        // -(instancetype _Nonnull)initWithResponseType:(AKFResponseType)responseType __attribute__((objc_designated_initializer));
+        [Export("initWithResponseType:")]
+        [DesignatedInitializer]
+        IntPtr Constructor(ResponseType responseType);
+
+        // -(void)cancelLogin;
+        [Export("cancelLogin")]
+        void CancelLogin();
+
+        // -(void)logOut;
+        [Export("logOut")]
+        void LogOut();
+
+        // -(void)logOut:(AKFLogoutHandler _Nullable)handler;
+        [Export("logOut:")]
+        void LogOut([NullAllowed] LogoutHandler handler);
+
+        // -(void)requestAccount:(AKFRequestAccountHandler _Nonnull)handler;
+        [Export("requestAccount:")]
+        void RequestAccount(RequestAccountHandler handler);
+
+        // -(UIViewController<AKFViewController> * _Nonnull)viewControllerForEmailLogin;
+        [Export("viewControllerForEmailLogin")]
+        //[Verify(MethodToProperty)]
+        //ViewController ViewControllerForEmailLogin { get; }
+        UIViewController GetViewControllerForEmailLogin();
+
+        // -(UIViewController<AKFViewController> * _Nonnull)viewControllerForEmailLoginWithEmail:(NSString * _Nullable)email state:(NSString * _Nullable)state;
+        [Export("viewControllerForEmailLoginWithEmail:state:")]
+        UIViewController GetViewControllerForEmailLogin([NullAllowed] string email, [NullAllowed] string state);
+
+        // -(UIViewController<AKFViewController> * _Nonnull)viewControllerForPhoneLogin;
+        [Export("viewControllerForPhoneLogin")]
+        //[Verify(MethodToProperty)]
+        //ViewController ViewControllerForPhoneLogin { get; }
+        UIViewController GetViewControllerForPhoneLogin();
+
+        // -(UIViewController<AKFViewController> * _Nonnull)viewControllerForPhoneLoginWithPhoneNumber:(AKFPhoneNumber * _Nullable)phoneNumber state:(NSString * _Nullable)state;
+        [Export("viewControllerForPhoneLoginWithPhoneNumber:state:")]
+        UIViewController GetViewControllerForPhoneLogin([NullAllowed] PhoneNumber phoneNumber, [NullAllowed] string state);
+
+        // -(UIViewController<AKFViewController> * _Nullable)viewControllerForLoginResume;
+        [NullAllowed, Export("viewControllerForLoginResume")]
+        //[Verify(MethodToProperty)]
+        //ViewController ViewControllerForLoginResume { get; }
+        UIViewController GetViewControllerForLoginResume();
+    }
+
+    interface IAccountKit { }
```

Comment:
This interface is not necessary due `AccountKit` will be generated as a class and not as an interface because it has no `[Protocol]` attribute.